crates/common/types/blobs_bundle.rs

```diff
@@ -63,6 +63,19 @@ pub fn blob_from_bytes(bytes: Bytes) -> Result<Blob, BlobsBundleError> {
     Ok(buf)
 }
 
+pub fn bytes_from_blob(blob: Bytes) -> [u8; BYTES_PER_BLOB * 31 / 32] {
```

Comment:
Abstract this value into a constant `BYTES_PER_BLOB * 31 / 32` and add some comment on it.

Reply:
Done [380e1bc](https://github.com/lambdaclass/ethrex/pull/2204/commits/380e1bc3383e7a30b2ad4b3247f5c5774249668a)